lib/libutee/include/pta_system.h

```diff
@@ -23,4 +23,17 @@
  */
 #define PTA_SYSTEM_ADD_RNG_ENTROPY	0
 
+/*
+ * Derives a device and TA unique key. The caller can also provide extra data
+ * that will be mixed together with existing device unique properties. If no
+ * extra data is provided, then the derived key will only use device unique
+ * properties.
```

Comment:
`will only use device unique properties` ... and caller TA UUID.

Reply:
Thanks, will fix!